augurpy/estimators.py

```diff
@@ -21,13 +21,16 @@ def _raise_exception(exception_message: str):
     raise Exception(exception_message)
 
 
+params = Params()
+
+
 def create_estimator(
     classifier: Union[
         Literal["random_forest_classifier"],
         Literal["random_forest_regressor"],
         Literal["logistic_regression_classifier"],
     ],
-    params: Params,
+    params: Params = params,
```

Comment:
You could also set params to None 
Then an if clause in create_estimator which creates a Params object if params is None.

Reply:
Thats a great tip! Thanks. Then there is no random variable floating around :+1